tests/cupy_tests/core_tests/test_reduction.py

```diff
@@ -51,8 +51,9 @@ def test_shape4(self):
         self.check_int8_sum((512 + 1, 256 * 256 + 1), axis=1)
 
     def test_shape5(self):
-        self.check_int8_sum((4 * 1024 * 1024, 1), axis=1)
-        self.check_int8_sum((4 * 1024 * 1024, 1), axis=0)
+        size = 2 << 32 // cupy.core.core.simple_reduction_function._block_size
```

Comment:
Shouldn't it be `(2 << 30) // cupy.core.core.simple_reduction_function._block_size` in order to get the same size?